saleor/graphql/discount/mutations.py

```diff
@@ -101,12 +101,15 @@ class VoucherInput(graphene.InputObjectType):
     min_amount_spent = Decimal(
         description="Min purchase amount required to apply the voucher."
     )
+    min_quantity_of_products = graphene.Int(
+        description="Minimal quantity of products required to apply the voucher."
```

Comment:
This has to be a precise description. Minimal quantity of products but where? How is it calculated when I have the same product but twice? I think it would be more precise if we said:

```suggestion
        description="Minimal quantity of checkout items required to apply the voucher."
```

Reply:
Another question is whether the field name `min_quantity_of_products` is accurate enough? Maybe it should be called `min_checkout_items_quantity`?

Reply:
Sure, I'm working on it.